app/views/machines/show.html.erb

```diff
@@ -1 +1,8 @@
 <h1><%= @machine.location %> Vending Machine</h1>
+
+<h2>Snacks</h2>
+<% @machine.snacks.each do |snack| %>
+  <%= snack.name %>: $<%= snack.price %>0
```

Comment:
anytime you print out a price, you might want it to look like an actual dollar amount. You could possibly create a model method to format it, but there's also an action view helper called `number_to_currency` that you can use for this formatting as well.